### ISSUE TITLE
Potential Issue with 60 Day Receivables

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1399]
In GitLab by @tim.traylor on Mar 7, 2019, 07:04

**Submitted by:** "Cheryl Hamelin" <cheryl.hamelin@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2019-03-07-53613

**Server:** Internal

**Client/Site:** Multiple

**Account:** NA

**Issue:** 

From ticket http://www.servicedesk.answernet.com/profiles/ticket/2019-03-07-53613 from Cheryl Hamelin

Good Morning,

I am hoping you can check something for me. I am working on the monthly Corporate A/R Summary Report for Rich Soltoff, Cori and Gary. Out of the 17 sites below that I have completed, only Orlando has an amount in the 60 day column. In the 4 ½ years I have been doing this, it is the first time this has happened. Please check and see if there is something wrong with the report. Rich needs the report this morning and I do not want to give him incorrect numbers.

* Allentown
* Atlanta/Jonesboro/Aurora
* Billerica/Inbound
* Billings
* Chicago
* Columbus
* Bismark
* Hayes
* El Paso
* Fair Oaks
* Fairlawn
* Lowell
* Memphis
* Billerica Outbound
* Orlando
* Phoenix Belize
* SAHosted

I appreciate your immediate assistance with this.

Regards,
Cheryl

----------------------------------------------------------------------------------------

In a nutshell, she is running the current billing cycle of the "Accounts Receivable As Invoiced".  For every site she listed except for Orlando, NONE of them have anything in the Age60 buckets.  This does seem like it should be almost a statistical impossibility.

Can you please check this ASAP?

Thanks,
Tim

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 7, 2019, 08:05

changed the description

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 7, 2019, 08:49

changed the description

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 7, 2019, 09:01

I wrote:

> After a few spot checks on the sites mentioned we were able to determine that this is due to the month of February only having 28 days in it. While the aging is based upon 30 days. This is why the aging that was in the 30 days on the previous invoice was not moved to the 60, as it had only been 58 days. Please keep in mind this would effect all sites that used the invoice date of 3/1/19 (or later) for their last invoice. Any site that has an invoice date in February should display aging in the 60 accordingly. 
> 
> Please note the billing cycle date and invoice dates are not always the same; but Aging and AR is based upon invoice date.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 7, 2019, 10:08

Cheryl wrote:

> Thank you so much for the information. It is much appreciated and very helpful. I also appreciate the quick turnaround.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 7, 2019, 10:08

assigned to @kdjstudios

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 12, 2019, 15:56

closed
